### ISSUE TITLE
a11y: Google Maps widget accessibility

### ISSUE THREAD

**Issue by ramiy** [pojome/elementor#3267]
## Description

The google maps widget need some accessibility love!

The current `iframe` should have an accessibility name, we can either add a `title` or an `aria-label` tag. The content of the tag will be the **Address** field.

![google-map-widget](https://user-images.githubusercontent.com/576623/35197681-933f456e-feeb-11e7-85d7-a633e372c7c2.png)

**Comment by macedvisioned** [comment]
Is there a solution for this issue, yet?
